### ISSUE TITLE
Help - Types Backward

### ISSUE THREAD

**Issue by BroKyle** [benweet/stackedit#835]
When I type in Stackedit, the type appears like this...?drawkcab gnipyt siht si yhW 
Of course it should read, "Why is this typing backward?"
What am I doing wrong?
Thanks.

**Comment by chicchera** [comment]
I have the same problem. I solved it using Firefox (or Palemoon)

**Comment by BroKyle** [comment]
Thanks alot. Any way to make it work with Chrome?

On Thu, Jul 2, 2015 at 3:33 PM, chicchera <notifications@github.com> wrote:

> I have the same problem. I solved it using Firefox (or Palemoon)
>
> —
> Reply to this email directly or view it on GitHub.
>



-- 
Bro. Kyle
Sign Up for Blog HERE <http://eepurl.com/RhsK1>

**Comment by benweet** [comment]
Did you change the "Right to left" setting?

**Comment by chicchera** [comment]
I did: nothing happens.

**Comment by BroKyle** [comment]
Yes. I even went back and forth changing it, thinking that may make it
work. Thanks.

On Thu, Jul 2, 2015 at 4:23 PM, chicchera <notifications@github.com> wrote:

> I did: nothing happens.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benweet/stackedit/issues/835#issuecomment-118172138>.
>



-- 
Bro. Kyle
Sign Up for Blog HERE <http://eepurl.com/RhsK1>

**Comment by BroKyle** [comment]
Oh, by the way, it didn't. I have been using Stackedit for a year or two
jut fine, on this same computer. It just started this yesterday.

On Thu, Jul 2, 2015 at 4:30 PM, Kyle Morris <deut649@gmail.com> wrote:

> Yes. I even went back and forth changing it, thinking that may make it
> work. Thanks.
>
> On Thu, Jul 2, 2015 at 4:23 PM, chicchera <notifications@github.com>
> wrote:
>
>> I did: nothing happens.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/benweet/stackedit/issues/835#issuecomment-118172138>.
>>
>
>
>
> --
> Bro. Kyle
> Sign Up for Blog HERE <http://eepurl.com/RhsK1>
>
>
>
>
>


-- 
Bro. Kyle
Sign Up for Blog HERE <http://eepurl.com/RhsK1>

**Comment by chicchera** [comment]
Oh, I am so glad: I started using it yesterday. It worked for a short while and then stopped.

**Comment by MarcusHallettUAH** [comment]
This has happened from time to time. As much as it pains me to say it (my Tech Ethics professor is probably frowning), the problem has always self-corrected when the browser was restarted. Also certain browser extensions interact poorly with stackedit.

**Comment by BroKyle** [comment]
Thank you.

On Fri, Jul 31, 2015 at 9:48 AM, Marcus Hallett <notifications@github.com>
wrote:

> [image: Boxbe] <https://www.boxbe.com/overview> Marcus Hallett (
> notifications@github.com) is not on your Guest List
> <https://www.boxbe.com/approved-list?tc_serial=22147953539&tc_rand=844979384&utm_source=stf&utm_medium=email&utm_campaign=ANNO_MWTP&utm_content=001&token=To%2FwmenbJPZiauWjrWqQBxVEA73ZhaExJDIfgwCSIOL%2Fp5dvJN6o9VtK9q%2FJcuWC&key=O9vqrQTCu6Zb0bP3VmeK495WXE5dOHkKK6TdCOE8dxE%3D>
> | Approve sender
> <https://www.boxbe.com/anno?tc_serial=22147953539&tc_rand=844979384&utm_source=stf&utm_medium=email&utm_campaign=ANNO_MWTP&utm_content=001&token=To%2FwmenbJPZiauWjrWqQBxVEA73ZhaExJDIfgwCSIOL%2Fp5dvJN6o9VtK9q%2FJcuWC&key=O9vqrQTCu6Zb0bP3VmeK495WXE5dOHkKK6TdCOE8dxE%3D>
> | Approve domain
> <https://www.boxbe.com/anno?tc_serial=22147953539&tc_rand=844979384&utm_source=stf&utm_medium=email&utm_campaign=ANNO_MWTP&utm_content=001&dom&token=To%2FwmenbJPZiauWjrWqQBxVEA73ZhaExJDIfgwCSIOL%2Fp5dvJN6o9VtK9q%2FJcuWC&key=O9vqrQTCu6Zb0bP3VmeK495WXE5dOHkKK6TdCOE8dxE%3D>
>
> This has happened from time to time. As much as it pains me to say it (my
> Tech Ethics professor is probably frowning), the problem has always
> self-corrected when the browser was restarted. Also certain browser
> extensions interact poorly with stackedit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benweet/stackedit/issues/835#issuecomment-126713501>.
>
>


-- 
Bro. Kyle
Sign Up for Blog HERE <http://eepurl.com/RhsK1>

**Comment by chicchera** [comment]
Well, thanks. I've been a programmer for over 40 years and completely empathize...
